Configure.cmake

```diff
@@ -78,49 +98,46 @@ CHECK_C_SOURCE_COMPILES("
 
 set (CMAKE_REQUIRED_FLAGS ${FLAGS_ENABLE_FMA4})
 CHECK_C_SOURCE_COMPILES("
-#if defined(_MSC_VER)
-#include <intrin.h>
-#else
-#include <x86intrin.h>
-#endif
-int main() {
-  __m256d r = _mm256_macc_pd(_mm256_set1_pd(1), _mm256_set1_pd(2), _mm256_set1_pd(3));
-}" COMPILER_SUPPORTS_FMA4)
+  #if defined(_MSC_VER)
+  #include <intrin.h>
```

Comment:
also indenting the code is not part of the refactoring. Please revert this chunk, here and in the other places I commented yesterday.